mubench.reviewsite/src/CSVHelper.php

```diff
@@ -0,0 +1,118 @@
+<?php
+namespace MuBench\ReviewSite;
+
+
+class CSVHelper
+{
+
+    public function createCSVFromStats($experiment, $stats)
+    {
+        $rows[] = ["sep=,"];
+        if (strcmp($experiment, "ex1") === 0) {
+            $rows[] =
+                ["detector", "project", "synthetics", "misuses", "potential_hits", "open_reviews", "need_clarification",
+                    "yes_agreements", "no_agreements", "total_agreements", "yes_no_agreements", "no_yes_agreements",
+                    "total_disagreements", "kappa_p0", "kappa_pe", "kappa_score", "hits", "recall"];
+        } elseif (strcmp($experiment, "ex2") === 0) {
+            $rows[] = ["detector", "project", "potential_hits", "open_reviews", "need_clarification", "yes_agreements",
+                "no_agreements", "total_agreements", "yes_no_agreements", "no_yes_agreements", "total_disagreements",
+                "kappa_p0", "kappa_pe", "kappa_score", "hits", "precision"];
+        } else {
+            $rows[] = ["detector", "project", "misuses", "potential_hits", "open_reviews", "need_clarification",
+                "yes_agreements", "no_agreements", "total_agreements", "yes_no_agreements", "no_yes_agreements",
+                "total_disagreements", "kappa_p0", "kappa_pe", "kappa_score", "hits", "recall"];
+        }
+        foreach ($stats as $key => $stat) {
+            $columns = [];
+            $columns[] = $stat->getDisplayName();
+            $columns[] = $stat->number_of_projects;
+            if (strcmp($experiment, "ex1") === 0) {
+                $columns[] = $stat->number_of_synthetics;
+            }
+            if (strcmp($experiment, "ex1") === 0 || strcmp($experiment, "ex3") === 0) {
+                $columns[] = $stat->number_of_misuses;
+            }
+            $kappa_p0 = round($stat->getKappaP0(), 3);
+            $kappe_pe = round($stat->getKappaPe(), 3);
+            $kappa_score = round($stat->getKappaScore(), 3);
+            if (strcmp($experiment, "ex2") === 0) {
+                $recall = round($stat->getPrecision() * 100, 1);
```

Comment:
We do round and *100 for presentation purposes in the Web frontend, but this is data export, so we should export the original, precise value. Same goes for recall below.